addons/website_event_track/controllers/event_track.py

```diff
@@ -543,3 +544,42 @@ def _get_dt_in_event_tz(self, datetimes, event):
             utc.localize(dt, is_dst=False).astimezone(timezone(tz_name))
             for dt in datetimes
         ]
+
+    @http.route('/event/send_email_reminder', type="jsonrpc", auth="public", website=True)
+    def send_email_reminder(self, track_id, email_to):
+        template = self.env.ref("website_event_track.email_reminder", raise_if_not_found=False)
+        if not template:
+            return {'success': False, 'error': 'missing_template'}
+
+        valid_email_to = tools.email_normalize(email_to)
+        if not track_id or not valid_email_to:
+            return {'success': False, 'message': _('Invalid data.')}
+        request.session['website_event_track.email_reminder'] = valid_email_to
+        track = request.env['event.track'].sudo().browse(track_id)
```

Comment:
Shoudln't we at least check the track exists, is published, and linked to a future or ongoing event ? Otherwise it looks like a route to poll all potential tracks and leak db info about what is still not published ^^ .